kinoml/datasets/kinomescan/pkis2.py

```diff
@@ -0,0 +1,109 @@
+from collections import defaultdict
+
+import pandas as pd
+
+from .utils import KINOMEScanMapper
+from ..core import BaseDatasetProvider
+from ...core.protein import AminoAcidSequence
+from ...core.ligand import Ligand
+from ...core.measurements import PercentageDisplacementMeasurement
+from ...core.conditions import AssayConditions
+from ...utils import datapath, defaultdictwithargs
+
+
+class PKIS2DatasetProvider:
```

Comment:
* Should subclass `object` at the very least
* Should likely subclass a `KinomeScanDatasetProvider` base class, which should provide a variety of useful methods